MonoGame.Framework/Audio/OpenAL.cs

```diff
@@ -175,7 +175,13 @@ public enum EfxEffectType
 
     public class AL
     {
+#if IOS || MONOMAC
+        public const string NativeLibName = "/System/Library/Frameworks/OpenAL.framework/OpenAL";
+#elif ANDROID
+        public const string NativeLibName = "libopenal32.so";
+#else
         public const string NativeLibName = "soft_oal.dll";
+#endif
```

Comment:
we should probably split these out into partial classes. Get rid of these pesky #if blocks